presto-docs/src/main/sphinx/security/authorization.rst

```diff
@@ -0,0 +1,130 @@
+=============
+Authorization
+=============
+
+Presto can be configured to enable authorization support for HTTP endpoints.
+
```

Comment:
Before getting into server configuration - lets move the section below where you cover the roles, above.